spacy/tests/pipeline/test_textcat.py

```diff
@@ -871,3 +872,43 @@ def test_textcat_multi_threshold():
 
     scores = nlp.evaluate(train_examples, scorer_cfg={"threshold": 0})
     assert scores["cats_f_per_type"]["POSITIVE"]["r"] == 1.0
+
+
+def test_save_activations():
+    fix_random_seed(0)
```

Comment:
I don't think we need to fix the random seed here (same for `test_save_activations_multi`)

Reply:
Removed fixing of the random seed in both cases.